### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by jimflores5**
My biggest worries on this chapter are:
1) We left out a major idea.
2) The content misses accepted JavaScript best practices for modules.

### CODE REVIEW

File: src/chapters/modules/exporting.rst [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+import them into ``index.js``.
+
+Exporting a Single Function
+----------------------------
+
+Let's start by exporting the ``isPalindrome`` function. At the bottom of the
+``practiceExports.js`` code, add the line ``module.exports = isPalindrome;``.
+This makes the function available to other files.
+
+In ``index.js`` we import ``practiceExports.js`` with a ``require`` statement.
```
Comment (gildedgardenia) [inline src/chapters/modules/exporting.rst]: "In ``index.js``, ..."

File: src/chapters/modules/wrap-up.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+========
+
+In this chapter, we showed how to use ``require`` to pull a module into your
+project, and we presented two ways to use ``module.exports``. Of course, these
+are not the only ways to share content.
+
+A quick search online shows that besides functions, we can also share
+individual variables. There are also alternative syntaxes for
+``module.exports`` - even one that exports as an object, but imports as a
+function (which means no dot notation). :-O
```
Comment (gildedgardenia) [inline src/chapters/modules/wrap-up.rst]: The emoji should be eliminated. Screen readers cannot process it.

File: src/chapters/modules/studio.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
+Studio: Combating Imposter Syndrome
+====================================
+
+There is a widely recognized condition called **Impostor Syndrome**. First
+described in the 1970s, it refers to a situation where someone doubts their
+accomplishments, and they fear that their success is the result of "faking it".
+People experiencing imposter syndrome ignore external evidence of their skills,
+and they attribute their success to luck.
+
+At this point in their learning journey, many LC101 students feel doubt about
```
Comment (gildedgardenia) [inline src/chapters/modules/studio.rst]: Remove mention of LC101 so content can be used in other programs.

File: src/chapters/modules/studio.rst [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+-----------
+
+Take a few moments in the studio to consider, share, and discuss the following:
+
+#. Have you ever felt the effects of Imposter Syndrome? When?
+#. Have you ever responded to a compliment by diminishing the work that earned
+   you the praise? If so, why did you answer in that way?
+#. How do you feel in a test/quiz/studio when someone finishes much earlier
+   than you?
+#. What are you most proud of from your time working in LC101?
```
Comment (gildedgardenia) [inline src/chapters/modules/studio.rst]: Remove mention of LC101 so content can be used in other programs.

File: src/chapters/modules/exporting.rst (hunk 1 above)
Comment (jimflores5) [reply src/chapters/modules/exporting.rst]: Fixed.

File: src/chapters/modules/wrap-up.rst (hunk 2 above)
Comment (jimflores5) [reply src/chapters/modules/wrap-up.rst]: OK. :-(

File: src/chapters/modules/studio.rst (hunk 3 above)
Comment (jimflores5) [reply src/chapters/modules/studio.rst]: Done.

File: src/chapters/modules/studio.rst (hunk 4 above)
Comment (jimflores5) [reply src/chapters/modules/studio.rst]: Done.

**Pull request merged**